### DISCUSSION TITLE
Default theme borders adding color does not work

### DISCUSSION THREAD

**Discussion by carlanthonyjoven** [chakra-ui/chakra-ui#5058] [Q&A] [answered]
Here is a code snippet of what I wanted to do. I tried to add color but it does not work. Please help, I am trying to set a global border color for all. Thanks in advance.

```javascript
const theme = extendTheme({
  borders: {
    '2px': '2px solid gray.200'
  }
});
```

**Comment by primos63** [comment, marked as answer]
The shorthand form for `border` is not parsed for Chakra colors. So `border='2px solid gray.200'` will not work even as a prop. You can specify any of the regular colors, 'red', 'green', 'lightgray', a hex value or rgb(). If you want the color value from the theme you can do 
```javascript
import { extendTheme } from "@chakra-ui/react";

import { getColor } from "@chakra-ui/theme-tools"
import { theme as defaultTheme } from "@chakra-ui/theme"

const borderColor = getColor(defaultTheme, 'gray.200', 'gray')

const theme = extendTheme({
  borders: {
    '2px':  `2px solid ${borderColor}`
  }
}
```

**Reply by carlanthonyjoven** [reply]
Thank you! @primos63. I appreciate your very helpful and quick response.

**Reply by StKostyantin** [reply]
Now getColor is deprecated, what are the alternatives?

**Reply by thesyntaxninja** [reply]
Wondering the same thing. What is the workaround now?

**Reply by primos63** [reply]
The function won't be removed until the next major release, v3.  If you moved from v1 to v2 then from what I can [see](https://chakra-ui.com/docs/styled-system/style-props#borders) there's a `borderColor` prop now. You should use that.

If you still want the getColor functionality but don't want to use the deprecated function, then you can always roll your own.
Something like this would work (it's a simplistic solution):
```javascript
const getColor = (theme, color, fallback) => {
  let colorExists = false
  const chakraColor = color.split('.')

  if (theme.colors.hasOwnProperty(chakraColor[0])) {
    if (theme.colors[chakraColor[0]].hasOwnProperty(chakraColor[1])) {
      colorExists = true
    }
  }

  return colorExists ? 
    theme.colors[chakraColor[0]][chakraColor[1]] : 
    theme.colors[fallback]['500']
};
```
